Automattic/wp-calypso
# Pattern Library: Change CTA URLs to `/setup/assembler-first`

Fixes https://github.com/Automattic/dotcom-forge/issues/6250

## Proposed Changes

I've updated the relevant CTA URLs to `/setup/assembler-first` instead of `/start`.

| Header | All about patterns section |
| - | - |
| ![all about patterns](https://github.com/Automattic/wp-calypso/assets/1101677/da1102c6-e402-4093-aa8e-640faff12620) | ![header](https://github.com/Automattic/wp-calypso/assets/1101677/e886db10-d313-4816-9de8-523c32c6f297) |

To be clear, I did **NOT** update this CTA. I think `/start/account/user` is quite suitable in that context.

![create account](https://github.com/Automattic/wp-calypso/assets/1101677/c9a3fb69-d943-463c-a58d-d1ce6bab3443)


## Testing Instructions



1. Be logged in
2. Navigate to `/patterns`
3. Ensure that the `Get started` link in the header leads to `/setup/assembler-first`
4. Open `/patterns` in an incognito window
5. Ensure that the `Get started` link in the header leads to `/setup/assembler-first`
6. Scroll down to the `All about patterns` section
7. Ensure that the `Build a site` button leads to `/setup/assembler-first`
